generation/ou_struct.go

```diff
@@ -0,0 +1,89 @@
+package generation
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/organizations"
+	"github.com/aws/aws-sdk-go-v2/service/organizations/types"
+)
+
+// --- OU ----------------------------------------------------------------------
+// OU is a struct that represents an OU in the AWS Organizations structure.
+// It can be used to represent the entire structure or a substructure in the
+// style of a tree.
+type OU struct {
+	Id       string
+	Name     string
+	Children []*OU
+	Accounts []types.Account
+}
+
+// addChildren adds the given OUs to the OU's children slice.
+func (o *OU) addChildren(children []*OU) {
```

Comment:
Oh wow this is now giving me serious Java vibes... Getters and setters? What a blast!

Reply:
(Realised I highlighted the wrong part of the code there, but I was referring to all the other getters in that file)

Reply:
Private setters public getters. Just makes creating other parts of the program a bit more clear.